packages/react-components/src/index.ts

```diff
@@ -364,6 +364,18 @@ export {
   usePortal_unstable,
 } from '@fluentui/react-portal';
 export type { PortalProps, PortalState } from '@fluentui/react-portal';
+
+export {
```

Comment:
What do you think of exporting this from `unstable` for now? This would help facilitate a final coherence pass over all the inputs before publishing them. (`Input` is still in `unstable` currently.)

Reply:
Whatever will be exported by `react-components` and not in `unstable` will end up in RC

Reply:
moved to unstable, 100% good with that